sourcecode/hub/resources/views/components/content/search.blade.php

```diff
@@ -25,5 +22,83 @@ class="btn-secondary"
             <x-icon name="search" />
             {{ trans('messages.search') }}
         </x-form.button>
+
+        <button
```

Comment:
The filter should not be in the search file.
This code is very similar to the filter in the sidebar, maybe that component can be re-used, or split into smaller components that can be re-used.